pkgs/crane-utils/src/bin/crane-resolve-workspace-inheritance.rs

```diff
@@ -420,6 +452,9 @@ mod tests {
             [target.'cfg(unix)'.dependencies]
             unix = { version = "unix-vers" , features = ["some"] }
 
+            [lints.rust]
```

Comment:
I'm not sure what's up with the placement of these, by the way, but apparently this is deterministic

Reply:
Not sure either but as long as its deterministic it's fine by me!